<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VaultFace</class>
 <widget class="QMainWindow" name="VaultFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color:#EEEEEE;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QWidget" name="panel_top" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>4</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2C2C2C;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#2C2C2C;
color: #fcd33e;</string>
         </property>
         <property name="text">
          <string>Vault Face</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
         <property name="indent">
          <number>13</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_data">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #4caf50;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #4caf20;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
         </property>
         <property name="text">
          <string>Ajouter Donn√©e</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_vault">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #4caf50;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #4caf20;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
         </property>
         <property name="text">
          <string>Ajouter Dossier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="panel_left" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #252526;</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="panel_right" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1E1E1E;
font-size:16px;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_data">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">margin-bottom:4px;</string>
              </property>
              <property name="text">
               <string>Dossier parent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_parent">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>33</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;
height:38px;
border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">margin-bottom:4px;</string>
              </property>
              <property name="text">
               <string>Nom de la donn√©e</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_name">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>33</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;
height:38px;
border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">margin-top:10px;
margin-bottom:4px;</string>
              </property>
              <property name="text">
               <string>Type de chiffrement</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="select_cyfer">
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;
height:38px;
border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
              <property name="minimumContentsLength">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>AES</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>RSA</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ChaCha20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blowfish</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DES </string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">margin-top:10px;
margin-bottom:4px;</string>
              </property>
              <property name="text">
               <string>Valeur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_value">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>33</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;
height:38px;
border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QPushButton" name="btn_modify">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: #4caf50;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #4caf20;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
                </property>
                <property name="text">
                 <string>Valider</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_delete">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: #f44336;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #f14316;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
                </property>
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>75</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_folder">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lbl_folder">
              <property name="styleSheet">
               <string notr="true">margin-bottom:4px;</string>
              </property>
              <property name="text">
               <string>Nom du dossier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_folder">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>33</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;
height:38px;
border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>44</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QPushButton" name="btn_save_folder">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: #4caf50;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #4caf20;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
                </property>
                <property name="text">
                 <string>Valider</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_delete_folder">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	background-color: #f44336;
	border-radius:8px;
	color:white;
	height:38px;
	font-size:16px;
	
}

QPushButton:hover{
	background-color: #f14316;
	border-radius:8px;
	height:38px;
	font-size:16px;
	
}</string>
                </property>
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>451</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_folders">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QListView" name="listView">
              <property name="styleSheet">
               <string notr="true">background-color: #333;

color:white;

border-radius:5px;
border: 1px solid #AAA;
padding: 0 8;</string>
              </property>
              <property name="viewMode">
               <enum>QListView::ViewMode::IconMode</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_datas">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTableWidget" name="tableDatas">
              <property name="styleSheet">
               <string notr="true">border-radius:5px;
border: 1px solid #AAA;
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
